extensions/amp-consent/0.1/consent-config.js

```diff
@@ -149,8 +151,56 @@ export class ConsentConfig {
         }
       }
     }
+    return this.mergeGeoOverride_(config).then(mergedConfig =>
+      this.validateMergedGeoOverride_(mergedConfig)
+    );
+  }
 
-    return config;
+  /**
+   * Merge correct geoOverride object into toplevel config.
+   * @param {?JsonObject} config
+   * @return {!Promise<?JsonObject>}
```

Comment:
would it ever return a promise of null?

can change to:
`!Promise<!JsonObject>`

Reply:
Yup, you're right